### ISSUE TITLE
Update in setup.cfg: license-file -> license_files

### ISSUE THREAD

**Issue by martenwa** [google/python-fire#423]
This happens both in version 0.5.0 and in master:
```
 * QA Notice: setuptools warnings detected:
 * 
 * 	The license_file parameter is deprecated, use license_files instead.
 * 	Usage of dash-separated 'license-file' will not be supported in future versions. Please use the underscore name 'license_file' instead
 ```

**Comment by dbieber** [comment]
Sounds good. Let's do the rename.

**Comment by hugovk** [comment]
In fact, it can be omitted entirely because setuptools automatically includes `LICENSE`:

> A list of glob patterns for license related files that should be included. If neither license_file nor license_files is specified, this option defaults to `LICEN[CS]E*`, `COPYING*`, `NOTICE*`, and `AUTHORS*`.

https://setuptools.pypa.io/en/latest/references/keywords.html#keywords

Please see PR https://github.com/google/python-fire/pull/434.

**Comment by dbieber** [comment]
Closed by #434
